xinference/model/llm/core.py

```diff
@@ -152,6 +152,7 @@ def __init__(
         llm_spec: "LLMSpecV1",
         quantization: Optional[str],
         model_path: Optional[str] = None,
+        multimodal_projector: Optional[str] = None,
```

Comment:
This seemed not used? This options may be illustrated in `to_dict` and then we can expose this option to users on web UI.